src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -224,6 +224,17 @@ trait WorkflowComponent {
       findWorkflowsBySubmissionId(submissionId).filter(_.status inSet(statuses.map(_.toString))).result
     }
 
+    def countWorkflowsByQueueStatus: ReadAction[Map[String, Int]] = {
+      findQueuedAndRunningWorkflows.result.map { recs =>
+        val grouped = recs.groupBy {
+          // lump all of the workflows currently being handled by Cromwell into "Active"
+          case runningRecord if WorkflowStatuses.runningStatuses.map(_.toString).toSet contains runningRecord.status => "Active"
```

Comment:
terminal statuses are not particularly informative of anything, will be large and may make the query slower. The only one I care about is the last one though. If the query is not slower then I don't really care.